src/schema/README.md

```diff
@@ -742,6 +742,7 @@ rule is applied.
     | `selectors`          | List of expressions; any evaluating false indicate rule does not apply                                         |
     | `columns`            | Object with keys that may be found in `objects.columns`, values either a requirement level or an object        |
     | `initial_columns`    | An optional list of columns that must be the first N columns of a file                                         |
+    | `index_columns`      | A list of columns that uniquely identify a row.                                                                |
```

Comment:
@effigies is this optional or required?  Lines [182](https://github.com/effigies/bids-specification/blob/769c62e5ca63c5d3602ffce3d8b5fd10ec5a763b/tools/schemacode/bidsschematools/render/tables.py#L182) and [218](https://github.com/effigies/bids-specification/blob/769c62e5ca63c5d3602ffce3d8b5fd10ec5a763b/tools/schemacode/bidsschematools/render/tables.py#L218) seem to imply it, if so 
```suggestion
    | `index_columns`      | An optional list of columns that uniquely identify a row.                                                                |
```